pages/login.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<html lang=" en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Harshna Haswani">
+  <meta name="description" content="Test yourself and grow into a rockstar developer">
+  <title> RockStar Dev — Login</title>
+
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="login ">
+
+  <header>
+    <h1 class="heading"><a href="/">RockStar Dev</a></h1>
+    <h2 id="page-title">Login</h2>
+  </header>
+  <main class="main-content login">
+    <form class="form" action="quizzes.html">
+      <div class="form-item status-plain-outline">
```

Comment:
is there a way that you can use directly the input tag without wrapping it, it works fine, but in the future, it will become a mess.

Reply:
There are style based on the wrapper so I can't remove it now